### ISSUE TITLE
Hotkeys not working on macOS (Chrome, Safari)

### ISSUE THREAD

**Issue by roelvangils** [SSAgov/ANDI#65]
Thanks for the wonderful tool. I've recentely started to integrate ANDI in my manual accessibility tests, and it's a tremendous help. I especially like the 'laser'.

One thing I couldn't get to work, are the hotkeys. Alt is Option on macOS, I know that, but none of the combinations seem to work.

```
alt+
= Relaunch
. Next Element
, Prev Element
/ Active Element
' ANDI Output
` Section Jump
```

**Comment by JohnCotterSSA** [comment]
We're glad you're finding ANDI to be wonderful!

We addressed this issue in Version 27.0.3. ANDI is now detecting the mac platform. When it does, it will indicate the key combination necessary to execute the ANDI hotkeys in the hotkeys menu (keyboard icon on the upper right). For your information, the key combination on Macs is Ctrl + Alt (option) + Key.

Please let us know if you come across any additional issues with hotkeys and please specify the browser and OS.

**Comment by shekita88** [comment]
- [ ] Direct deposit 💵 [**@**]()HatcherFamilyst
